CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# How to Contribute to CasterSoundboard
+
+CasterSoundboard is a volunteer-run project, built and maintained by many contributors. We welcome new team members, and value feedback from users using CastrerSoundboard.
+
+The following is a set of guidelines for contributing to CasterSoundboard, which are hosted in the Jupiter Broadcasting Organization on GitHub. These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+### Reporting a Bug Report or Requesting a Feature
+We utilize a template to keep our issue tracker organized. Please fill it out and remove areas that are not related to your Bug Report or Feature Request. Also keep these guidelines in mind as well:
+
+* Search before posing to avoid duplicates
+* Please keep the topic of the issue or feature request to **ONE** problem or request.
```

Comment:
Ah, just re-read #53 and saw that it's more about what library to use if the feature goes ahead, probably not the best example but I still think we should allow somewhat generic PR's for the moment